### ISSUE TITLE
Intermittent spec failure when testing with webpacker

### ISSUE THREAD

**Issue by deivid-rodriguez** [activeadmin/activeadmin#7290]
### Did you find a bug?

An intermittent spec failure.

### Expected behavior

Specs pass consistently.

### Actual behavior

```
Failures:

  1) ActiveAdmin::ResourceController::PolymorphicRoutes polymorphic_url with multiple belongs_to (not fully supported yet) within the posts page works with a category as parent
     Failure/Error: javascript_tag = active_admin_namespace.use_webpacker ? javascript_pack_tag(path) : javascript_include_tag(path)

     ActionView::Template::Error:
       767: unexpected token at ''
     # ./lib/active_admin/views/pages/base.rb:42:in `block (2 levels) in build_active_admin_head'
     # ./lib/active_admin/views/pages/base.rb:41:in `block in build_active_admin_head'
     # ./lib/active_admin/views/pages/base.rb:27:in `build_active_admin_head'
     # ./lib/active_admin/views/pages/base.rb:9:in `build'
     # ./app/views/active_admin/resource/index.html.arb:3:in `block in __home_runner_work_activeadmin_activeadmin_app_views_active_admin_resource_index_html_arb__2517167330308779720_47007235259800'
     # ./app/views/active_admin/resource/index.html.arb:1:in `new'
     # ./app/views/active_admin/resource/index.html.arb:1:in `__home_runner_work_activeadmin_activeadmin_app_views_active_admin_resource_index_html_arb__2517167330308779720_47007235259800'
     # ./lib/active_admin/resource_controller/streaming.rb:13:in `index'
     # ./spec/unit/resource_controller/polymorphic_routes_spec.rb:19:in `block (4 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     # JSON::ParserError:
     #   767: unexpected token at ''
     #   ./lib/active_admin/views/pages/base.rb:42:in `block (2 levels) in build_active_admin_head'

Finished in 33.13 seconds (files took 4.71 seconds to load)
674 examples, 1 failure
```

### How to reproduce

Hard, for now.

**Comment by tagliala** [comment]
Is it possible that this issue is caused by the following line?

https://github.com/activeadmin/activeadmin/blob/1b6dd5c2c5722f896ca36c3fa447b4db63038446/tasks/test_application.rb#L81

In any case, it should be fixed, the proper path is "gemfiles/rails_61_webpacker/Gemfile"

**Comment by deivid-rodriguez** [comment]
Not sure, maybe, let's fix it yeah.

**Comment by tagliala** [comment]
Experienced this issue here: https://github.com/activeadmin/activeadmin/runs/5108572235?check_suite_focus=true

**Comment by javierjulio** [comment]
I had to disable the webpacker build in CI temporarily because it now always fails but I’m not sure what the cause or fix is. Any help would be appreciated. The build is commented out in our CI file.

**Comment by tagliala** [comment]
@javierjulio please take a look at #7851

Probably this issue is caused by the fact that webpacker is old and uses old dependencies

The ideal would be to switch to shakapacker and Rails 7

Also, I have some other improvements for the CI that I can submit in different PRs

PS: any idea why the code coverage upload is failing?

**Comment by javierjulio** [comment]
@tagliala thank you. I've approved that PR and just rebased it now that the Codecov migration in #7853 has been merged. Yes, you're very likely right about the secrets/code coverage upload part. I was pretty adamant though on doing the Codecov migration anyway and having been half way through it I wasn't going to stop. Sorry for the noise there.

The webpacker build is now passing! Great catch. I wouldn't have thought it would be related to an earlier Node version. I'll merge that and close this issue. As always thank you for your help and patience. 🙏🏻
